front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/adjustConfWithTrainToModify.ts

```diff
@@ -34,58 +18,67 @@ function convertStepToPointOnMap(step?: ArrayElement<Path['steps']>): PointOnMap
     : undefined;
 }
 
-export function loadPathFinding(path: Path, dispatch: Dispatch) {
+export function loadPathFinding(
+  path: Path,
+  dispatch: Dispatch,
+  osrdActions: operationalStudiesConfSliceActionsType | stdcmConfSliceActionsType
+) {
   if (path.steps && path.steps.length > 1) {
-    dispatch(updatePathfindingID(path.id));
-    dispatch(updateItinerary(path));
-    dispatch(updateOrigin(convertStepToPointOnMap(path.steps[0])));
-    dispatch(updateDestination(convertStepToPointOnMap(path.steps.at(-1))));
+    dispatch(osrdActions.updatePathfindingID(path.id));
+    dispatch(osrdActions.updateItinerary(path));
+    dispatch(osrdActions.updateOrigin(convertStepToPointOnMap(path.steps[0])));
+    dispatch(osrdActions.updateDestination(convertStepToPointOnMap(path.steps.at(-1))));
```

Comment:
Could you destructure all needed actions first ?

Reply:
For me, it's better to give 1 arg than 4

Reply:
Good observation